src/hooks/useSyncChainQuery.ts

```diff
@@ -27,6 +27,7 @@ export default function useSyncChainQuery() {
 
   const urlChainId = getParsedChainId(parsedQs)
   const previousUrlChainId = usePrevious(urlChainId)
+  const [nextChainId, setNextChainId] = useState<number | undefined>(undefined)
```

Comment:
nit: `nextUrlChainId` to be clear about the relationship

Reply:
it's really not the `nextUrlChainId` though, it's just updating the chain id to what was in the url param

Reply:
maybe move it down to be defined alongside `previousChainId`? putting it here makes it look like it might have been misnamed.

Reply:
good point